src/main/java/io/jenkins/plugins/report/jtreg/main/comparator/Arguments.java

```diff
@@ -8,11 +8,17 @@ public static Options parse(String[] arguments) {
         if (arguments.length >= 1) {
             for (int i = 0; i < arguments.length; i++) {
                 String currentArgument = arguments[i];
+                boolean hadHyphen = false; // for enforcing at least one hyphen
 
                 // delete all leading - characters
                 while (currentArgument.charAt(0) == '-') {
+                    hadHyphen = true;
```

Comment:
the regex would do better :)

Reply:
I redid it to use regex for replacing and checking the leading hyphens. In later commit will come the change to using an Argument class in the arguments parsing